openedx/core/djangoapps/appsembler/sites/api.py

```diff
@@ -270,3 +270,13 @@ def post(self, request, format=None):
 class CustomDomainView(CreateAPIView):
     queryset = AlternativeDomain.objects.all()
     serializer_class = AlternativeDomainSerializer
+
+
+class CompileSassView(APIView):
+    def post(self, request, format=None):
+        site_uuid = request.data.get('site_uuid')
+        org = Organization.objects.get(edx_uuid=site_uuid)
+        site = org.sites.get()
+        configuration = SiteConfiguration.objects.get(site=site)
+        configuration.compile_microsite_sass()
```

Comment:
This is needed so `SiteConfiguration` model uses the service adapter.

```suggestion
        configuration.init_api_client_adapter(site)
        configuration.compile_microsite_sass()
```